sunpy/map/mapbase.py

```diff
@@ -1572,6 +1578,31 @@ def draw_rectangle(self, bottom_left, width: u.deg, height: u.deg, axes=None, **
         else:
             axes_unit = self.spatial_units[0]
 
+        if width is not None or height is not None:
+            warnings.warn(SunpyPendingDeprecationWarning("Use of `width` and `height` is depreciated and these will be removed in future"))
```

Comment:
```suggestion
            warnings.warn(SunpyPendingDeprecationWarning("Use of `width` and `height` is deprecated and will be removed in future"))
```
You can even raise a simple `DeprecationWarning` here.

Reply:
I believe a `SunpyPendingDeprecationWarning` is more apt.

Reply:
I believe a `SunpyPendingDeprecationWarning` is more apt here.